packages/imperative/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## `8.8.4`
+
+- BugFix: Broken validation of positional arguments. [#2384](https://github.com/zowe/zowe-cli/pull/2384)
+- Enhancement: Added a test to validate it . [#2384](https://github.com/zowe/zowe-cli/pull/2384)
```

Comment:
```suggestion
```
I believe we don't need to add an entry for non-user facing items like a new regression test 😋